### ISSUE TITLE
new name for stack?

### ISSUE THREAD

**Issue by light-matters** [vukics/Wigner_Time#19]
I'm wondering about the pros and cons of renaming `stack` with `queue` or some such?

The reasoning is that this function is used to build an experiment (i.e. at the user layer), whereas `stack` is really describing the implementation.

**Comment by light-matters** [comment]
Other ideas:  `procedure`, `design` (nouns), `stage`, `build` (verbs)

**Comment by light-matters** [comment]
It's been stewing in the back of my mind a bit and I like the idea of `build`.
```python
tl.build(
tl.create(...),
cl.ramp(...),
...
)
```

**Comment by vukics** [comment]
I do not think that the implementation has much to do with `stack`, at least I didn't have the implementation in mind when I coined the name.

I'm not sure about `build`, in fact, `create` also "builds" in itself.

Stack has the connotation for me of something layered, which is what I meant.

**Comment by vukics** [comment]
Probably we should not change the names of basic functions any more, as so much depends on them, even in notebooks.
